Apps/phgsgmail/gsgmail.json

```diff
@@ -792,6 +792,400 @@
+                {
+                    "data_path": "action_result.data.*.snippet",
+                    "data_type": "string",
+                    "example_values": [
+                        "Your Google Workspace monthly invoice is available. Please find the PDF document attached at the bottom of this email. IMPORTANT: The balance will be automatically charged so you don&#39;t need to take"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.parsed_plain_body",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.data.*.sizeEstimate",
+                    "data_type": "numeric",
+                    "example_values": [
+                        55260
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.threadId",
+                    "data_type": "string",
+                    "example_values": [
+                        "1792b1cd66228bf2"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.labelIds",
+                    "data_type": "string",
+                    "example_values": [
+                        "INBOX"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.parsed_html_body",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.data.*.id",
+                    "data_type": "string",
+                    "example_values": [
+                        "1792b1cd66228bf2"
+                    ]
+                },
+                {
+                    "data_path": "action_result.summary.total_messages_returned",
+                    "data_type": "numeric",
+                    "example_values": [
+                        1
+                    ]
+                },
+                {
+                    "data_path": "summary.total_objects",
+                    "data_type": "numeric",
+                    "example_values": [
+                        1
+                    ]
+                },
+                {
+                    "data_path": "summary.total_objects_successful",
+                    "data_type": "numeric",
+                    "example_values": [
+                        1
+                    ]
+                }
+            ],
+            "versions": "EQ(*)",
+            "render": {
+                "width": 2,
+                "height": 5,
+                "type": "custom",
+                "title": "Get Email",
+                "view": "gsgmail_view.get_email"
+            }
         }
-    ]
-}
\ No newline at end of file
+    ],
+    "directory": "gsuiteforgmail_9c73f233-2c4a-406a-855e-41d8d2497d0e",
+    "version": 1,
+    "appname": "-",
+    "executable": "spawn3",
+    "disabled": false
+}
```

Comment:
From the title of MR, this is just to add a new action `get email`.

Why do we need to add these changes here?

Reply:
So this is actually copied from a PR from the PS team. I am not sure what this content is actually; I assumed maybe the output script generated it